var/spack/repos/builtin/packages/nim/package.py

```diff
@@ -132,6 +130,22 @@ def append_rpath(path, libdirs):
         if spec.satisfies("+sqlite"):
             append_rpath("lib/wrappers/sqlite3.nim", spec["sqlite"].libs.directories)
 
+        # Musl defines SysThread as a struct *pthread_t rather than an unsigned long as glibc does.
+        if self.spec.satisfies("^[virtuals=libc] musl"):
+            if self.spec.satisfies("@devel,1.9.3:"):
```

Comment:
Not sure where 1.9.3 is coming from... Not a tag listed in https://github.com/nim-lang/Nim/tags, but it's in between 1.6.20 and 2, so it doesn't matter I guess.

Reply:
Could be:
```suggestion
            if self.spec.satisfies("@devel,2:"):
```
ref: https://github.com/nim-lang/Nim/commit/1dab8ba334a1b53f3c8780bc8a4f8045a6ea0375

Reply:
It's for the packaged in-dev version `@1.9.3`.
https://github.com/spack/spack/blob/cb9c9e6b8ce78c32609920c65072ffe344d92d63/var/spack/repos/builtin/packages/nim/package.py#L38